pkg/services/ngalert/metrics/multi_org_alertmanager.go

```diff
@@ -133,11 +226,28 @@ func (a *AlertmanagerAggregatedMetrics) Describe(out chan<- *prometheus.Desc) {
 	out <- a.silencesQueryDuration
 	out <- a.silencesPropagatedMessagesTotal
 	out <- a.silences
+
+	out <- a.dispatchAggrGroups
+	out <- a.dispatchProcessingDuration
 }
 
 func (a *AlertmanagerAggregatedMetrics) Collect(out chan<- prometheus.Metric) {
 	data := a.registries.BuildMetricFamiliesPerTenant()
 
+	data.SendSumOfCountersPerTenant(out, a.numNotifications, "alertmanager_notifications_total", metrics.WithLabels("integration"), metrics.WithSkipZeroValueMetrics)
```

Comment:
For reviewers: `WithSkipZeroValueMetrics` means that a metric is not emitted if it's value would be zero. For notifications, this is very common (e.g. a user is not using a particular integration), and so doing this saves emitting lots of metrics unnecessarily, particularly in systems with many orgs.